core/computed/entity-classification.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {makeComputedArtifact} from './computed-artifact.js';
+import {NetworkRecords} from './network-records.js';
+import {Util} from '../util.cjs';
+import UrlUtils from '../lib/url-utils.js';
+import thirdPartyWeb from '../lib/third-party-web.js';
+
+/** @typedef {Map<string, LH.Artifacts.Entity>} EntityCache */
+
+class EntityClassification {
+  /**
+   * @param {EntityCache} entityCache
+   * @param {string} url
+   * @return Entity | undefined
```

Comment:
I can't find your `@return`/`@returns` question now, but the reason we use it is because [google/closure compiler code uses it](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#return-type-description). Unclear why, maybe someone long ago liked how it mimicked `return` statements more than the more prose-like description? Things we pass down in code long after what required their use has been removed and forgotten :)

Reply:
That makes sense. Re: `return`/`returns`, I found that we have both variations in our code already, and thus initiated a rename of all to `returns` in a branch, which seemed trivial. However, while trying to set the preferences as a lint rule, via 

```
"prefer": {
  "returns": "returns",
}
```

I found that we've disabled `jsdoc` validation via `'require-jsdoc': 0,` in `.eslintrc`. Do you have further context to why it's disabled? (Also bringing all comments to jsdoc valid, plus migrating to recommended eslint plugin, seemed non-trivial, so I've stashed it for now).

Reply:
> I found that we've disabled jsdoc validation via 'require-jsdoc': 0, in .eslintrc. Do you have further context to why it's disabled?

I think it differed too much from google/jscompiler style and so was annoying, but that was a very very long time ago -https://github.com/GoogleChrome/lighthouse/pull/15 :)